examples/diesel-mysql/main.rs

```diff
@@ -1,10 +1,32 @@
 use diesel::{
-    connection::Connection, mysql::MysqlConnection, query_dsl::RunQueryDsl,
-    sql_query, sql_types, QueryableByName,
+    connection::Connection, mysql::MysqlConnection, sql_query, sql_types,
+    QueryableByName, RunQueryDsl,
 };
 use jiff::civil;
 use jiff_diesel::ToDiesel;
 
+mod schema {
+    diesel::table! {
+        datetimes {
+            id -> Integer, // Diesel tables require an ID column.
```

Comment:
Technically you can go without an `id` column, you just need to declare a random column to be the "primary key" via `datetimes(t) { …}` (turns the `t` column in your fake primary key). That works as long as you write these `table!` calls manually and don't relay on the fact that the primary key is actually a primary key.